### ISSUE TITLE
Wrong key mapping of < > on latinamerican keyboard layout

### ISSUE THREAD

**Issue by FloValdes** [pqrs-org/Karabiner-Elements#3030]
Latinamerican keyboards have the < > key next to the Z. When Karabiner is running this key changes to | °, even without any configuration.

Maybe it's related to [#3002](https://github.com/pqrs-org/Karabiner-Elements/issues/3002) ?

**Comment by MuhammedZakir** [comment]
> Maybe it's related to #3002 ?

Yes. This has been a bug for a long time. It's unlikely to be fixed soon. Swapping or changing that key using simple modification usually "fixes" this problem.

**Comment by rebeluca** [comment]
Same issue for me, with italian keyboard #2634 . I cannot find the symbols to create a simple modification.

**Comment by MuhammedZakir** [comment]
> Same issue for me, with italian keyboard #2634 . I cannot find the symbols to create a simple modification.

Hopefully, this will work for you:

1. Quit Karabiner-Elements.
2. Remove and plug-in your keyboard again.
3. Open Karabiner-EventViewer.
4. Open an on-screen keyboard viewer.
5. Check for the characters you want in the keyboard viewer and press the corresponding key in your keyboard.
6. EventViewer should show correct keycodes.

Note: sometimes you may not see the characters you want without pressing `option`, but using that is usually a bad idea. Restrict the modifier to only `shift` like you would type normally. And sometimes, you might have to change the keyboard layout.
